File: C#/Base_Functions.cs
private static string StandardKey(string Key)
{
    var ret = Regex.Replace(Key, "[^a-zA-Z0-9]", "");
    return ret.Substring(0,16);
}

private static string Encrypt(string input, string key)
{
    byte[] inputArray = Encoding.UTF8.GetBytes(input);
    AesCryptoServiceProvider AES = new AesCryptoServiceProvider();
    string _Key = StandardKey(key);
    AES.KeySize = 256;
    AES.BlockSize = 128;
    AES.Mode = CipherMode.ECB;
    AES.Padding = PaddingMode.PKCS7;
    AES.GenerateKey();
    AES.Key = Encoding.ASCII.GetBytes(_Key);
    ICryptoTransform cTransform = AES.CreateEncryptor();
    byte[] resultArray = cTransform.TransformFinalBlock(inputArray, 0, inputArray.Length);
    AES.Clear();
    return Convert.ToBase64String(resultArray, 0, resultArray.Length);
}

private string GetSignedData(string _toHash , string PrivateKey)
{
    string _SignData = Encrypt(_toHash, PrivateKey);
    return _SignData;
}

File: Python/Base_Functions.py
import re
import base64
from Crypto.Cipher import AES
from pkcs7 import PKCS7Encoder
def standard_key(key):
   return re.sub(r'\W+', '', key)[:16]

def Encrypt(input_string, key):
   _key = standard_key(key)
   input_string = input_string.encode(encoding="utf-8")
   aes = AES.new(standard_key(_key), AES.MODE_ECB)
   aes.block_size = 128
   return base64.b64encode(aes.encrypt(PKCS7Encoder().encode(input_string)))

def GetSignedData(_toHash , PrivateKey):
   _SignData = Encrypt(_toHash, PrivateKey)
   return _SignData